vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -1327,20 +1296,19 @@ void vtkF3DRenderer::ShowMetaData(bool show)
   if (this->MetaDataVisible != show)
   {
     this->MetaDataVisible = show;
+    this->UIActor->SetMetaDataVisibility(show);
     this->MetaDataConfigured = false;
-    this->RenderPassesConfigured = false;
     this->CheatSheetConfigured = false;
```

Comment:
do you still need the "configured" logic for the UI elements ?

Reply:
hum, i would need to check but I got a feeling the logic may be simplifiable now

Reply:
sure, take a look and let me know

Reply:
ConfigureTextActor will be able to go once dropzone is a ImGUI stuff as well.

Reply:
the rest need to stays afaict